biojava-structure/src/test/java/org/biojava/nbio/structure/secstruc/TestBasePairParameters.java

```diff
@@ -0,0 +1,31 @@
+import org.biojava.nbio.structure.secstruc.BasePairParameters;
+import org.junit.Test;
+
+/**
+ * Created by luke on 7/21/17.
+ */
+public class TestBasePairParameters {
```

Comment:
It's an interesting idea.  Right now it's mainly the removeDups flag that's relevant in that method.  Some X-ray structures contain nearly-identical units, and so removeDups intends to only count the first instance of a helix of interest.  

The usage of this code at the moment is to run through RCSB and extract parameters of unique sequence/protein combinations for about 4000 structures, for only X-ray structures. 
 There are some notable NMR structures that are very accurate but they haven't been considered yet, because many among them are not as reliable [in the opinion of some people... I think they're alright].  The speed is amazing with all your caching, so I can see the effect of changing criteria in a manner of a minute or two.

So I reworked the constructors so different criteria can be provided (whether its DNA or RNA and whether to removeDups, and just the generic version for DNA that takes a Structure), and at some point RNA/DNA hybrid helices might be considered. 

So I added commits to make the changes, but let me know if you think anything else could be optimized and commented.